docs/flowcollector-flows-netobserv-io-v1beta1.adoc

```diff
@@ -617,20 +624,122 @@ Required::
 | `string`
 | Address of the Kafka server
 
+| `sasl`
+| `object`
+| SASL authentication configuration. [Unsupported (*)].
+
 | `tls`
 | `object`
-| TLS client configuration. When using TLS, verify that the address matches the Kafka port used for TLS, generally 9093. Note that, when eBPF agents are used, the Kafka certificate needs to be copied in the agent namespace (by default it is `netobserv-privileged`).
+| TLS client configuration. When using TLS, verify that the address matches the Kafka port used for TLS, generally 9093.
 
 | `topic`
 | `string`
 | Kafka topic to use. It must exist, NetObserv will not create it.
 
+|===
+== .spec.exporters[].kafka.sasl
+Description::
++
+--
+SASL authentication configuration. [Unsupported (*)].
+--
+
+Type::
+  `object`
+
+
+
+
+[cols="1,1,1",options="header"]
+|===
+| Property | Type | Description
+
+| `clientIDReference`
+| `object`
+| Reference to the secret or config map containing the client ID
+
+| `clientSecretReference`
+| `object`
+| Reference to the secret or config map containing the client secret
+
+| `type`
+| `string`
+| Type of SASL authentication to use, or `DISABLED` if SASL is not used
+
+|===
+== .spec.exporters[].kafka.sasl.clientIDReference
+Description::
++
+--
+Reference to the secret or config map containing the client ID
+--
+
+Type::
+  `object`
+
+
+
+
+[cols="1,1,1",options="header"]
+|===
+| Property | Type | Description
+
+| `file`
+| `string`
+| File name within the config map or secret
+
+| `name`
+| `string`
+| Name of the config map or secret containing the file
+
+| `namespace`
+| `string`
+| Namespace of the config map or secret containing the file. If omitted, assumes same namespace as where NetObserv is deployed. If the namespace is different, the config map or the secret will be copied so that it can be mounted as required.
```

Comment:
```suggestion
| Namespace of the config map or secret containing the file. If omitted, the default is to the same namespace as where Network Observability Operator is deployed. If the namespace is different, the config map or the secret is copied so that it can be mounted as required.
```